static/index.html

```diff
@@ -16,43 +16,79 @@
 
 <html>
   <head>
+    <link type="text/css" rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:400,300,600,700&display=swap">
     <script src="/fhirclient/fhir-client.min.js"></script>
     <script src="/jquery/jquery.min.js"></script>
+    <script src="language-assets.js"></script>
+    <link rel="stylesheet" href="assets/styles.css">
     <script>
       $(function() {
         FHIR.oauth2.ready()
-          .then(client => handle(client.user, client.encounter))
+          .then((client) => 
+          {
+            $("#language-selector").on("change", () => { 
+              setLanguage($('#language-selector').val());
+            });   
+            if(!isAvailable(client.encounter) || !isAvailable(client.encounter.id)) { 
+              showError('#error-no-encounter');
+            } else {
+              var userResourceType;
+
+              // Older FHIR server that doesn't support id_token and therefore the fhirUser property.
+              if(!isAvailable(client.user) || !isAvailable(client.user.resourceType)) {
```

Comment:
Do we need isAvailable?  I would think a simple truth test here is sufficient?

Reply:
The smart on FHIR client was populating null rather than undefined for certain properties under some conditions (like client.user.resourceType was returned as null when running in test mode unless I am misremembering) when data was populated with "default values". So rather than attempt to make the distinction and determine all possible values (when in test mode vs. when partially populated with data from an older FHIR server vs. when populated with latest FHIR server standard data) I thought it best to just check for both conditions as both seemed possible.

Reply:
I am not a javascript expert by any means, but my understanding is that undefined and null both evaluate to false...  What am I missing?